### PR TITLE
[zigbee2mqtt] Add Zigbee2mqtt Assistant

### PR DESCRIPTION
**Pull request by toxuin**
This adds Zigbee2mqtt-assistant, as per https://github.com/runningman84/helm-charts/issues/3

### CODE REVIEW

File: zigbee2mqtt/values.yaml [hunk 1]
```diff
@@ -54,6 +54,57 @@ persistence:
...
+  # pollingSchedule: "*/12 * * * *"
+  # netScanSchedule: "0 */3 * * *"
+
+  image:
+    repository: carldebilly/zigbee2mqttassistant
+    pullPolicy: IfNotPresent
+    tag: 0.3.175
+
+  mqtt:
+    server: mqtt.example.com
```
Comment (runningman84) [inline zigbee2mqtt/values.yaml:77]: Is localhost more appropriate here? It runs in the same network namespace anyway

File: zigbee2mqtt/templates/assistant-service.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{{- if .Values.assistant.enabled }}
+{{- $fullName := include "zigbee2mqtt.fullname" . -}}
+{{- $serviceName := printf "%s-%s" $fullName "assistant" -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ $serviceName }}
+  labels:
```
Comment (runningman84) [inline zigbee2mqtt/templates/assistant-service.yaml:8]: I think you can just use
{{- include "zigbee2mqtt.labels" . | nindent 4 }}

File: zigbee2mqtt/templates/assistant-ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{{- if and (.Values.assistant.enabled) (.Values.assistant.ingress.enabled) }}
+{{- $fullName := include "zigbee2mqtt.fullname" . -}}
+{{- $ingressPath := .Values.assistant.ingress.path -}}
+{{- $servicePort := .Values.assistant.service.port -}}
+{{- $serviceName := printf "%s-%s" $fullName "assistant" -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $serviceName }}
+  labels:
```
Comment (runningman84) [inline zigbee2mqtt/templates/assistant-ingress.yaml:10]: As below
{{- include "zigbee2mqtt.labels" . | nindent 4 }}

**Comment by runningman84** [comment]
@toxuin have you seen my comments?

**Comment by toxuin** [comment]
I’m sorry, I did not get any comment notifications. Where have you commented on this?

**Comment by runningman84** [comment]
You should see some comments as code review in the changes page

**Pull request merged**